### DISCUSSION TITLE
how to use unocss css

### DISCUSSION THREAD

**Discussion by Liripo** [pyloid/pyloid#8] [Q&A] [answered]
```js
import 'virtual:uno.css'

document.addEventListener('pyloidReady', function () {
  const app = createApp(App)
  app.use(ArcoVue).mount('#app')
})
```
Using `addEventListener` does not seem to be able to introduce tailwindcss or unocss.

**Comment by aesthetics-of-record** [comment]
I tested it.
vue3 + tailwindcss works.

I haven't tested unocss yet. But, I think it will probably all work.

### tailwindcss
[https://tailwindcss.com/docs/guides/vite#vue](https://tailwindcss.com/docs/guides/vite#vue)

#### main.ts
```ts
import { createApp } from 'vue'
import './style.css'
import App from './App.vue'

document.addEventListener('pyloidReady', () => {
  createApp(App).mount('#app')
})
```


This is the answer to the question as I understand it.
If you have another question, please answer with additional information.

**Comment by aesthetics-of-record** [comment, marked as answer]
I am currently developing create-pyloid-app cli tool.
Using it, you will be able to easily create templates for npm/yarn/pnpm/bun and react/vue/svelte.
